app/views/user_mailer/anonymous_or_unrevealed_notification.html.erb

```diff
@@ -0,0 +1,26 @@
+<% content_for :message do %>
+  <p><%= t(".greeting", user: style_bold(@user.default_pseud.byline)).html_safe %></p>
+
+  <p><%= t(".changed_status",
+           collection: style_link(@collection.title, collection_url(@collection)),
+           work: style_link(@work.title.html_safe, work_url(@work)),
+           status: @status).html_safe %></p>
```

Comment:
I'm checking with Translation on this. I know when it comes to numbers, they want us to remove interpolation and instead create a separate key depending on the possible value, but I'm not sure if they'll have issues with adjectives as well. If that's a problem for them, we'll need to change it to something more like:
```
<p><%= t(".changed_status.#{@status}",
            collection: style_link(@collection.title, collection_url(@collection)),
            work: style_link(@work.title.html_safe, work_url(@work)).html_safe %></p>

changed_status: 
  unrevealed: "The collection maintainers of %{collection} have changed the status of your work %{work} to unrevealed."
  anonymous: "The collection maintainers of %{collection} have changed the status of your work %{work} to anonymous."
  anonymous_unrevealed: "The collection maintainers of %{collection} have changed the status of your work %{work} to anonymous and unrevealed."
```

Reply:
Have you heard back from Translation about this?

Reply:
Not yet, but I've poked them again and will let you know when I hear back!

Reply:
The verdict is it's probably not _necessary_ to change it, but they'd find it neater, so if you could change it, that would be great!